### ISSUE TITLE
Feature request: support Scaleway DNS provider

### ISSUE THREAD

**Issue by francesco086** [qdm12/ddns-updater#660]
1. What's the feature?

Add support for [Scaleway](https://www.scaleway.com/) DNS provider.

2. Extra information?

Here [the link to its API](https://www.scaleway.com/en/developers/api/domains-and-dns/#path-records-update-records-within-a-dns-zone) to update the DNS records.

I would be happy to contribute, if someone could guide me a little.

**Comment by francesco086** [comment]
Hi @qdm12 !

It took me quite long (almost 1 year!) but I finally found the time to contribute.
Better late than never, right? Happy new year 😉 😜 

I tried on my local machine using docker and my scaleway account, and it works.

I only have issues with this error
```
ERROR setting initial up to date status: opening file: open /updater/data/updates.json: permission denied
```

But I have seen that this is a known issue, and it doesn't relate to my implementation.

PR: https://github.com/qdm12/ddns-updater/pull/899

**Comment by qdm12** [comment]
Happy new year! 🎆 

> I only have issues with this error

Yeah you need to change the ownership and/or permissions of the files you bind mount, usually this does it on your host:

```sh
chown -R 1000 data
chmod -R u+r+w+x data
```

**Comment by francesco086** [comment]
> Happy new year! 🎆
> 
> > I only have issues with this error
> 
> Yeah you need to change the ownership and/or permissions of the files you bind mount, usually this does it on your host:
> 
> ```shell
> chown -R 1000 data
> chmod -R u+r+w+x data
> ```

Yes, I read that in the documentation and used it, but it didn't help. Not sure if perhaps on mac OS something is different.

**Comment by Neustradamus** [comment]
@francesco086: Can you check for BookMyName (same company)?
- https://github.com/qdm12/ddns-updater/issues/241

Thanks in advance.

**Comment by Neustradamus** [comment]
Good news, your PR has been merged today by @qdm12! :)
- https://github.com/qdm12/ddns-updater/pull/899

Thanks for your work!
